web/app/js/components/Sidebar.jsx

```diff
@@ -104,29 +108,40 @@ class Sidebar extends React.Component {
     });
   }
 
-  toggleCollapse() {
-    if (this.state.initialCollapse) {
-      // fix weird situation where toggleCollapsed is called on pageload,
-      // causing the toggle states to be inconsistent. Don't toggle on the
-      // very first call to toggleCollapse()
-      this.setState({ initialCollapse: false});
-    } else {
-      this.setState({ collapsed: !this.state.collapsed });
+  handleNamespaceSelector(value) {
+    this.setState({namespaceFilter: value});
+  }
+
+  // Filters resources retrieved from a stat query by namespaces. Does not include resources of type "authority"
+  filterResourcesByNamespace(resources, namespace) {
```

Comment:
We could get rid of the resources we don't want to show when we process the metrics in `loadFromServer`. This could just be `delete resourceGroupings["authority"]` there, for example. This removes the need for the deleting of it in this method.

I'll also note that `services` aren't wired in, so you might want to handle that too.